foundation-extensions/identity-for-edge-network/api-reference.md

```diff
@@ -339,10 +339,19 @@ AEPIdentityItem *item = [[AEPIdentityItem alloc] initWithId:@"user@example.com"
 
 ## resetIdentities
 
-Clears all identities stored in the Identity extension and generates a new Experience Cloud ID \(ECID\) . Using this API does not remove the identifiers from the server-side User Profile Graph or Identity Graph.
+Clears all identities stored in the Identity extension and generates a new Experience Cloud ID \(ECID\). Using this API does not remove the identifiers from the server-side User Profile Graph or Identity Graph.
```

Comment:
Just as a heads up, but you don't need to include the \\ in front of the ( or ). It'll still render fine.

Reply:
Thank you! I've removed the backslashes